ext/spl/spl_fixedarray.stub.php

```diff
@@ -6,10 +6,8 @@ class SplFixedArray implements IteratorAggregate, ArrayAccess, Countable, JsonSe
 {
     public function __construct(int $size = 0) {}
 
-    /**
-     * @tentative-return-type
-     * @deprecated
-     */
+    /** @tentative-return-type */
+    #[\Deprecated(since: '8.4', message: 'the __unserialize() method must be overloaded instead')]
```

Comment:
```suggestion
    #[\Deprecated(since: '8.4', message: 'the __unserialize() method must be overridden instead')]
```

Overloaded is the wrong term: https://www.php.net/manual/en/language.oop5.overloading.php. “Overridden” matches `#[\Override]`.

Reply:
This is indeed better!